README.md

```diff
@@ -61,6 +61,12 @@ COPY libcrypteia.so /opt/lib/libcrypteia.so
 ENV LD_PRELOAD=/opt/lib/libcrypteia.so
 ```
 
+If you are using Python you will need to add our Crypteia python package to the PYTHONPATH in order for things to "just work". The result of this will be that `os.environ["MYVAR"]`, `os.environ.get("MYVAR")`, and `os.getenv("MYVAR")` will be routed to the `getenv` system call and therefore take advantage of the Crypteia rust extension.
```

Comment:
Suggest using `SECRET` to be consistent with docs.

```suggestion
If you are using Python you will need to add our Crypteia python package to the PYTHONPATH in order for things to "just work". The result of this will be that `os.environ["SECRET"]`, `os.environ.get("SECRET")`, and `os.getenv("SECRET")` will be routed to the `getenv` system call and therefore take advantage of the Crypteia rust extension.
```